### PR TITLE
add devtools section to responsive basics doc

### PR DESCRIPTION
**Pull request by kaycebasques**
@petele I added a section to your "responsive web design basics" doc on using DevTools to view how a page looks under different media queries. Looks good to you?

FYI I also cleaned up the accompanying DevTools sections that I link off to while I was at it.

### CODE REVIEW

File: src/content/en/tools/chrome-devtools/device-mode/index.md [hunk 1]
```diff
@@ -20,15 +24,26 @@ Use Chrome DevTools' Device Mode to build mobile-first, fully responsive website
 * Evaluate your site's performance using the [network emulator](/web/tools/chrome-devtools/profile/network-performance/network-conditions), without affecting traffic to other tabs.
 * Accurately [simulate device input](/web/tools/chrome-devtools/iterate/device-mode/device-input-and-sensors) for touch events, geolocation, and device orientation
 
-## How to disable Device Mode
+## Toggle Device Mode {: #toggle }
```
Comment (petele) [inline src/content/en/tools/chrome-devtools/device-mode/index.md]: Section titles should be sentence case.

File: src/content/en/tools/chrome-devtools/index.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ into Google Chrome. Use the DevTools to iterate, debug and profile your site.
 
 Dogfood: Looking for the latest, [Chrome Canary](https://tools.google.com/dlpage/chromesxs) always has the latest DevTools.
 
-## Opening the Chrome DevTools
+## Opening the Chrome DevTools {: #open }
```
Comment (petele) [inline src/content/en/tools/chrome-devtools/index.md]: Sentence case

File: src/content/en/tools/chrome-devtools/index.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ into Google Chrome. Use the DevTools to iterate, debug and profile your site.
 
 Dogfood: Looking for the latest, [Chrome Canary](https://tools.google.com/dlpage/chromesxs) always has the latest DevTools.
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/index.md]: Can you please add {: .external} to all external links?

File: src/content/en/tools/chrome-devtools/index.md (hunk 2 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/index.md]: Ditto. This is also sentence case. "Chrome DevTools" always gets capitalized.

File: src/content/en/tools/chrome-devtools/index.md (hunk 3 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/index.md]: I wonder if we could handle this in the build phase...

**Pull request merged**